modules/ddl-and-loading/pages/modifying-a-graph-schema.adoc

```diff
@@ -28,6 +28,71 @@ Load or query operations which begin before the schema change will be completed
 Schema changes that are performed during times with high query or data loading volume may have a chance of failure.
 It is considered good practice to perform schema changes during times of low graph activity.
 
+== Schema Change Operations Best Practices
+
+Schema changes are essential for maintaining and improving your graph's structure. However, they must be done carefully to avoid impacting ongoing operations. 
+Following best practices minimizes the risk of failures and ensures smooth execution.
+
+=== Dos and Don'ts During Schema Changes
+
+*Do's:*
+
+* *Do ensure no active queries or loading jobs.* Run schema changes only after aborting ongoing jobs that might reference schema components being modified.
+* *Do perform schema changes during low-traffic periods.* This reduces the chances of failure and conflicts during schema modification.
+
+*Don'ts:*
+
+* *Don’t run queries or loading jobs during schema changes.* These jobs may fail if they interact with modified parts of the schema.
+* *Don’t change the schema during high-traffic periods.* Schema changes during heavy operations may cause system conflicts and errors.
+
+=== Low Traffic and High Traffic Periods
+
+Defining periods of low and high traffic is crucial to optimizing schema changes and reducing failure risks.
+
+*Low Traffic:*
+
+* Schema changes should be scheduled when there are minimal active operations (queries or data loads) to avoid conflicts.
+
+*High Traffic:*
+
+* Avoid schema changes during periods of high query or data load activity to prevent failure or inconsistent behavior.
+
+=== Risks of Running Jobs During Schema Changes
+
+Running queries or loading jobs while modifying the schema can lead to data inconsistency or errors. For instance, if a schema change modifies a vertex type and a query or loading job is still active, it will likely fail.
+
+*Best Practice:* Always abort active queries and loading jobs before performing a schema change.
+
+=== Verifying Schema Integrity After Changes
+
+Once the schema change is complete, verify data and schema consistency:
+
+*Verifying Data Consistency:*
+
+* Run the `gstatusgraph` command to check that the data size remains consistent with what was recorded before the schema change. This will ensure that no data loss or corruption has occurred during the schema update.
```

Comment:
Data loss and data corruption are some of the worst errors that a database can do.  Thing about it.
1. Shouldn't we recommend that users perform a backup before performing a schema change?
2. Check other database products to see if/when they mention "data loss", "data corruption", or "data inconsistency".  We want to see how they mention this.  
Compare these two: 
1. "It is essential to make sure that there are no loading jobs in progress when you start the schema change. Failure to follow this rule may result in data corruption." (It's the user's fault.)
2. "Schema changes sometimes result in data corruption." (It's TigerGraph's fault.)